deploy/k8s/standalone/default_quickstart_counter.yaml

```diff
@@ -0,0 +1,188 @@
+apiVersion: v1
```

Comment:
Better to put this into another folder like`deploy/k8s/sidecar`?

Reply:
done